insights/parsers/engine_log.py

```diff
@@ -1,37 +0,0 @@
-"""
```

Comment:
It's not allowed to remove any existing interface from insights-core v3.0, except for fixing bugs.

Your implementation to merge all corresponding engine logs to one parser file sounds good to me too, but you should keep it backward compatible.  E.g. leave an empty entry shell for the EngineLog in this `insights/parsers/engine_log.py` file, such as:

```python
from insights.parsers.ovirt_engine_log import EngineLog
```

Reply:
I like your suggestions to keep the file `engine_log.py` to ensure backward compatibility. I'm re-introducing this file and import statement. I made sure that no rule is using this parser before the changes were incorporated.

Reply:
Done